fmatch/matcher.py

```diff
@@ -7,12 +7,12 @@
 import logging
 
 # pylint: disable=import-error
-from elasticsearch import Elasticsearch
+from elasticsearch7 import Elasticsearch
```

Comment:
curious about this change?

Reply:
I think `elasticsearch7` can be installed using `elasticsearch==7.x.x`, this has been a previous issue for me too.

Reply:
yeah, if you don't specify the module version, the python interpreter just picks the latest one available